extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -147,11 +156,19 @@ export class LiveListManager {
   fetchDocument_() {
     let url = this.url_;
     if (this.latestUpdateTime_ > 0) {
-      url = addParamToUrl(
-        url,
-        'amp_latest_update_time',
-        String(this.latestUpdateTime_)
-      );
+      const parameters = this.enrolledInAppendRandomExperiment_
+        ? dict({
+            'amp_latest_update_time': String(this.latestUpdateTime_),
+            // AMP Caches do not always evict entries from their caches.
+            // This experiment adds a random identifier to reduce cache hits for enrolled documents.
+            'amp_random': String(
+              Math.floor(Math.random() * AMP_LIVE_LIST_MAX_RANDOM_NUMBER)
```

Comment:
Did we decide to not go with the `Date` header approach?

Reply:
The `Date` header approach needs additional testing on older browsers where our polyfill is providing a stubbed Response object.

Given the timing for the fix, and the remedy on the Google AMP Cache – I instead opted for the more narrow fix for enrolled documents.